resources/examples/prescriptionserver/Bundle-Response-App-GETChargeItemById.json

```diff
@@ -40,6 +40,10 @@
           {
             "system": "https://gematik.de/fhir/erp/NamingSystem/GEM_ERP_NS_PrescriptionId",
             "value": "200.086.824.605.539.20"
+          },
+          {
+            "system": "https://gematik.de/fhir/erp/NamingSystem/GEM_ERP_NS_AccessCode",
```

Comment:
Änderung bitte mit den Konsumenten abklären.

Reply:
Derzeit ist das in RU-DEV so umgesetzt, dass bei GET /ChargeItem/<id> das ChargeItem auch einen AccessCode hat.

Ich würde also diese Änderung mergen. Dann gleichen wir uns dem Bestehenden an.